cdk/kraken/src/deploy.ts

```diff
@@ -67,7 +92,7 @@ export class DeployJob extends CheckoutJob {
           name: "Deploy",
           run: dedent`aws eks --region us-east-1 update-kubeconfig --name production --role-arn arn:aws:iam::\${AWS_ACCOUNT_ID}:role/kubectl
 
-          # get repo name from synth step
+          # Get repo name from synth step
```

Comment:
Currently, after running `yarn tsc` and copying the local kraken to a labs product, the new workflow file generated has unnecessary slashes.

For example, `RELEASE_NAME=${{ steps.synth.outputs.RELEASE_NAME }}` becomes `RELEASE_NAME=\${{ steps.synth.outputs.RELEASE_NAME }}`, even though the code did not change.

Need to look into this before merging and see how to get rid of these slashes.

Reply:
It's because of the double dedent -_- fixed